tests/src/kakarot/instructions/test_push_operations.py

```diff
@@ -22,7 +23,27 @@ async def test__exec_push_should_raise(self, push_operations, i):
         with kakarot_error():
             await push_operations.test__exec_push_should_raise(i).call()
 
-    @pytest.mark.parametrize("i", range(1, 33))
+    @pytest.mark.parametrize("i", range(0, 33))
     async def test__exec_push_should_push(self, push_operations, i):
         res = await push_operations.test__exec_push_should_push(i).call()
         assert res.result.value == int_to_uint256(256**i - 1)
```

Comment:
back on this test few months after we wrote it, I think that it deserves a small comment to explain that because we fill the bytecode with `0xFF`, when we push _i_ bytes we have 0xFF..FF (i times), which is `256**i - 1`